backend/models/db/index-question-weights.model.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+module.exports = function indexQuestionWeights(sequelize, Sequelize, schema = 'public') {
```

Comment:
We can delete this file. We have moved all survey related things to the survey service